src/main/fc/fc_msp.c

```diff
@@ -944,6 +944,7 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
         sbufWriteU8(dst, rxConfig()->rx_spi_protocol);
         sbufWriteU32(dst, rxConfig()->rx_spi_id);
         sbufWriteU8(dst, rxConfig()->rx_spi_rf_channel_count);
+        sbufWriteU8(dst, rxConfig()->receiverType);
```

Comment:
Won't this cause problems if the receiver of the MSP response is expecting a specific length?

Reply:
Not if other party (configurator) checks for firmware version.